### ISSUE TITLE
Project status, DevTool

### ISSUE THREAD

**Issue by goodmind** [motorcyclejs/motorcyclejs#58]
Can you explain current status of project. 

What is @motorcyclets and why it's not in this repo?
How can I use Cycle.js DevTool with motorcycle?

**Comment by TylorS** [comment]
The current status of this repository is stable! It's been very reliable for me and my coworkers and it's not going anywhere.  

As for Cycle, there is no longer a strong tie between the two projects and they are likely to diverge more and more in the future. The Cycle Devtool was built for XStream and as far as I know will never be made to work with most as it's built using properties added to XStream for that purpose. 

As for Motorcycle.ts, I was hoping to keep it quiet for a while longer! I'm going to rebrand the project and reaffirm our focus on providing the best typed web framework TypeScript can possibly allow us to have. It's going to be a full rewrite and will introduce some new packages to capture how to architect and build real-life Motorcycle apps, with a large focus on better documentation.

**Comment by goodmind** [comment]
@TylorS Have you plans for motorcycle devtool?

**Comment by TylorS** [comment]
It's not my highest priority right now, but some other testing-related tools are. To be honest, I haven't seen a large amount of value from the devtool in it's current form, but if it's useful I'm sure it will get built eventually.

**Comment by cortopy** [comment]
is there something equivalent to https://github.com/cyclejs/react-native in the works?

**Comment by Frikki** [comment]
@cortopy 

> is there something equivalent to https://github.com/cyclejs/react-native in the works?

No, but it should be reasonably straightforward to implement. However, @TylorS and I are both very busy a.t.m., wherefore this is not a priority for us.

We are, of course, open to pull requests that implement this feature. If that is the case, we can open a separate issue to discuss it.
